config.py

```
@@ -1,8 +1,30 @@
+import os
 from pydantic_settings import BaseSettings
 from typing import Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)